packages/communication-react/api-extractor.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../../common/config/api-extractor/api-extractor.json",
-  "mainEntryPointFilePath": "<projectFolder>/dist/dist-esm/communication-react/src/index.d.ts"
+  "mainEntryPointFilePath": "<projectFolder>/dist/dist-esm/communication-react/src/api-summary.d.ts"
```

Comment:
The primary `communication-react.d.ts` we point to in the `types` field will need to not include the loaders, does this alter that?
We may need two build commands, one that generates the `api-summary.d.ts`, and one that generates the `communication-react.d.ts` and `loaders.d.ts`